Jellyfin.Server/Startup.cs

```diff
@@ -64,7 +63,6 @@ public void ConfigureServices(IServiceCollection services)
             app.UseResponseCompression();
 
             // TODO app.UseMiddleware<WebSocketMiddleware>();
-            app.Use(serverApplicationHost.ExecuteWebsocketHandlerAsync);
```

Comment:
Why?

Reply:
To me it seems simplest to have 1 delegate differentiate between websocket request and normal http request instead of having 2 that each ignore the other request type.

Reply:
But that's the middleware way. If a middleware doesn't apply to the current context, it calls the next one

Reply:
Ok, but then we should fully split them out, atm they are still being handled by the same instance of the same class

Reply:
Let's do it in another PR then